.github/workflows/testdata.yaml

```diff
@@ -0,0 +1,23 @@
+name: testdata
```

Comment:
Can we discuss the case scenarios:

The test-operator supports AllNamespaces, SingleNamespace, and OwnNamespace install modes.

- v1.0.0: is just the crd and the existing configmap resource
- v1.3.0: has the same crd as v1.0.0 and the updated configmap previously part of v2.0.0
- v2.0.0: has validating, mutating and conversion webhooks

The DELTA between v1.0.0 and v2.0.0 is clear
V2.0.0 should introduce webhooks 

But what is the DELTA between v1.0.0 and v1.3.0
What do we want to cover with v1.3.0? 
Do we want to add any changes that would not be a breaking change?
What scenarios do we need to cover?

Reply:
1.3.0 is just to harmonize with what's already there. It's something that can definitely be revisited. Right now, 2.0.0 includes this configmap. We can probably just collapse it to 1.0.0 and do the check. I think it's just a sanity check to make sure that annotations that include template syntax don't get affected.

Reply:
I think the list you put there is fine. I don't even think we need the 1.1.0 version. We only care about what is in the payload insofar as it interacts with OLM features/behavior. I.e. whether it has network policies, exposes metrics, uses HA, etc. if that isn't important for OLM, we don't need to add the complexity. When we get to things like probing the state of the installation, or want to test some boxcutter features etc, then I think this will become more necessary. For now, if it stamps out a configmap (or any other resource) just to say it stamped out a resource from the bundle, that's enough imo.

Reply:
>  I.e. whether it has network policies, exposes metrics, uses HA, etc. isn't important if it doesn't matter to OLM
It is important because we must to suppport those configs
We must ensure that OLM works well with NP, and etc

Reply:
> We must ensure that OLM works well with NP, and etc
I agree. But if working well means those resources are included when we create a bundle, I think unit tests would be sufficient to verify that a NetworkPolicy in the bundle gets properly rendered and that things that get renderer get applied to the cluster. 

If it means that OLM needs to interact with specific resources in some kind of special way (e.g. probing for status on specific resources), then I agree we'd probably want an e2e for that.